acs-integration-tests/src/test/java/com/ge/predix/acceptance/test/ACSAcceptanceIT.java

```diff
@@ -218,17 +189,20 @@ public void testCompleteACSFlow(final String endpoint, final HttpHeaders headers
 
     @DataProvider(name = "endpointProvider")
     public Object[][] getAcsEndpoint() throws Exception {
-        PolicyEvaluationRequestV1 policyEvalForBob = this.policyHelper.createEvalRequest("GET", "bob",
-                "/alarms/sites/sanramon", null);
+        PolicyEvaluationRequestV1 policyEvalForBob = this.policyHelper
+                .createEvalRequest("GET", "bob", "/alarms/sites/sanramon", null);
 
-        Object[][] data = new Object[][] {
-                { this.acsBaseUrl, this.headersWithZoneSubdomain, policyEvalForBob, "bob" } };
-        return data;
+        return new Object[][] { { this.acsBaseUrl, this.headersWithZoneSubdomain, policyEvalForBob, "bob" } };
     }
 
     @AfterClass
-    public void cleanUp() {
-        this.zoneHelper.deleteZone(this.acsAdminRestTemplate, this.acsZone1Name, this.registerWithZac);
+    public void tearDown() {
+        this.zoneHelper.deleteZone(this.acsAdminRestTemplate, this.acsZoneName, false);
+        this.uaaTestUtil.deleteClient(this.acsAdminRestTemplate.getResource().getClientId());
+
+        if (this.zacHelperUtil != null) {
+            this.zacHelperUtil.tearDown(this.acsZoneName);
```

Comment:
Want to keep the zone helper and the zacHelperUtil separate so that open source users do not have knowledge of ZAC